docs/.vuepress/config.js

```diff
@@ -106,6 +106,15 @@ module.exports = {
           ],
         },
       ],
+      "dev":[
+        {
+          title: "Dev",
+          collapsable: false,
+          children: [
+            "index"
```

Comment:
For only one document, we have new section. I think better to move this doc to zarb-go under contribution.md or find better place for taht.